nav2_system_tests/src/planning/test_planner_launch.py

```diff
@@ -29,17 +29,13 @@
 def main(argv=sys.argv[1:]):
     testExecutable = os.getenv('TEST_EXECUTABLE')
 
-    run_navfn = launch_ros.actions.Node(
-        package='nav2_navfn_planner',
-        node_executable='navfn_planner',
-        output='screen')
     run_lifecycle_manager = launch_ros.actions.Node(
         package='nav2_lifecycle_manager',
         node_executable='lifecycle_manager',
         node_name='lifecycle_manager',
         output='screen',
-        parameters=[{'node_names': ['navfn_planner']}, {'autostart': True}])
-    ld = LaunchDescription([run_navfn, run_lifecycle_manager])
+        parameters=[{'node_names': ['navfn_planner']}, {'autostart': False}])
```

Comment:
We don't need lifecycle manager in this test anymore, if I understand correctly. We should remove it.

Reply:
Yeah, that's part of my CI fixing issues.

Reply:
done